app/src/main/java/org/oppia/android/app/administratorcontrols/learneranalytics/ProfileAndDeviceIdFragmentPresenter.kt

```diff
@@ -38,16 +40,15 @@ class ProfileAndDeviceIdFragmentPresenter @Inject constructor(
   }
 
   private fun createRecyclerViewAdapter(): BindableAdapter<ProfileListItemViewModel> {
-    return BindableAdapter.MultiTypeBuilder
-      .newBuilder<ProfileListItemViewModel, ProfileListItemViewType> { viewModel ->
-        when (viewModel) {
-          is DeviceIdItemViewModel -> ProfileListItemViewType.DEVICE_ID
-          is ProfileLearnerIdItemViewModel -> ProfileListItemViewType.LEARNER_ID
-          is SyncStatusItemViewModel -> ProfileListItemViewType.SYNC_STATUS
-          else -> error("Encountered unexpected view model: $viewModel")
-        }
+    return multiTypeBuilderFactory.create<ProfileListItemViewModel,
```

Comment:
If we name the injected field as ``adapterFactory`` will this be able to fit on one line so that it doesn't need to be line-wrapped in this way?

Reply:
Yeah it should be able to.